src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped");
```

Comment:
The message in `stopWork` method should also be more informative. Consider changing it to 'Bulldozer has stopped working.' to clearly indicate the action and the type of machine.